### PR TITLE
Kong Gateway comparison table

### PR DESCRIPTION
**Pull request by mheap**
### Summary
Add Gateway feature comparison table from https://konghq.com/install#kong-community

### Testing
/gateway/latest/

### CODE REVIEW

File: app/_data/tables/features/gateway.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+columns:
+  - name: Open Source
+    cta: Get Started
+    cta_link: /
```
Comment (lena-larionova) [inline app/_data/tables/features/gateway.yml:4]: This just goes to the docs.konghq.com landing page - is that on purpose?

File: app/_data/tables/features/gateway.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      - name: Simple Data Transformations
+        tooltip: Add or remove headers, JSON data, or query strings 
+        oss: true
+        enterprise: true
+      - name: gRPC Transformations
+        tooltip: Translate requests from gRPC-Web and REST to backend gRPC services
+        oss: true
+        enterprise: true
+      - name: GraphQL
+        tooltip: Convert GraphQL queries to REST requests. Rate limit and cache GraphQL queries
```
Comment (lena-larionova) [inline app/_data/tables/features/gateway.yml:50]:
```suggestion
        tooltip: Convert GraphQL queries to REST requests. Rate limit and cache GraphQL queries.
```
If one has a period, then both should

File: app/_data/tables/features/gateway.yml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        oss: false
+        enterprise: true
+  - name: Security and Governance
+    items:
+      - name: Authentication
+        tooltip: Common methods of API authentication - Basic Auth, HMAC, JWT Key Auth, limited OAuth 2.0, limited LDAP
+        oss: true
+        enterprise: true
+      - name: Advanced Authentication
+        tooltip: Enterprise-grade API authentication - Full OAuth 2.0, OpenID Connect, Vault, Mutual TLS, JWT Signing/Resigning, full LDAP
```
Comment (lena-larionova) [inline app/_data/tables/features/gateway.yml:76]:
```suggestion
        tooltip: Enterprise-grade API authentication - Full OAuth 2.0, OpenID Connect, Vault, mutual TLS, JWT signing/resigning, full LDAP
```

File: app/_data/tables/features/gateway.yml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+      - name: Advanced Authorization (OPA)
+        tooltip: Control access to APIs with complex, programmable, enterprise-wide rules
+        oss: false
+        enterprise: true
+      - name: Secret Management
+        tooltip: Encrypt sensitive keys, certificates, and passwords
+        oss: false
+        enterprise: true
+      - name: FIPS 140-2 Compliance
+        tooltip: Generally Available version coming soon. Kong Gateway in a FIPS 140-2 compliant mode
```
Comment (lena-larionova) [inline app/_data/tables/features/gateway.yml:96]:
```suggestion
        tooltip: Generally available (GA) version coming soon. Kong Gateway in a FIPS 140-2 compliant mode.
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]
(forgot to set to request changes)

File: app/_data/tables/features/gateway.yml (hunk 1 above)
Comment (mheap) [reply app/_data/tables/features/gateway.yml:4]: Pending discussion, I forgot to mark as draft. It's likely to go to an install page, but pending decision

Comment (mheap) [reply app/_data/tables/features/gateway.yml:4]: Merging to use as a base for the new installation index, as this link will point there.

**Pull request merged**